user_guide/business_intelligence_reports.rst

```diff
@@ -6,9 +6,8 @@ Reports
 In OroCRM, you can create reports based on the details of any entity available in the system. 
 
 With the functionality, you can get easy-to-understand adjustable reports on the amount and activity of different 
-:ref:`customers <user-guide-basic-entities-customer-id>`, on statuses of 
-:ref:`carts or opportunities <user-guide-basic-sre>`, on performance of your 
-:ref:`marketing activity <user-guide-marketing>`, specifics of recent :ref:`activities <user-guide-activities-index>`, 
+customers, on statuses of carts or opportunities, on performance of your 
```

Comment:
Edit: customers, on the statuses of carts or opportunities, on the performance of your